packages/dds/shared-object-base/package.json

```diff
@@ -147,6 +147,11 @@
 		"typescript": "~5.3.3"
 	},
 	"typeValidation": {
-		"broken": {}
+		"broken": {
+			"RemovedFunctionDeclaration_bindHandles": {
```

Comment:
Why is this running on `@internal` things?  Oh well.

Reply:
I thought this seemed odd too, but it removed the entries from validateSharedObjectPrevious which seemed like the right thing to do.